spec/large_object_store_spec.rb

```diff
@@ -35,6 +35,18 @@ def delete(key)
   end
 end
 
+RSpec::Matchers.define :be_compressed do
+  match do |actual|
+    actual[LargeObjectStore::UUID_SIZE] == LargeObjectStore::COMPRESSED || actual[0] == LargeObjectStore::COMPRESSED
```

Comment:
some comment might be nice "# compression indicator is in first position for single page values and after the uuid for multi page values"